### PR TITLE
Latest devito

### PR DESCRIPTION
**Pull request by philippwitte**
Remove local copy of devito module and fetch judi

### CODE REVIEW

File: src/AzureBatch/docker/base_image/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN apt-get install -y htop vim git-core
 # Devito
 RUN apt-get install -y python3 python3-pip
 RUN pip3 install --upgrade pip
-RUN pip3 install --upgrade git+https://github.com/opesci/devito.git
+RUN pip3 install --upgrade git+https://github.com/devitocodes/devito.git
```
Comment (mloubout) [inline src/AzureBatch/docker/base_image/Dockerfile]: `pip3 install --upgrade devito` should be enough it's registered on PyPi now

File: src/AzureBatch/docker/tti_image/Dockerfile [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
-RUN pip3 install azure-storage
+RUN pip3 install azure-storage-blob==2.1.0 h5py segyio boto3
 
 # Current sympy bug -> force version 1.5
 RUN pip3 install sympy==1.5
 
-# Add application
-ADD ./tti /app/tti
+# Add devito setup from JUDI, keep only Python part
+RUN git clone --single-branch --branch examples https://github.com/slimgroup/JUDI.jl /app/JUDI.jl \
```
Comment (mloubout) [inline src/AzureBatch/docker/tti_image/Dockerfile:10]: The scripts above assume it installed in `~/.julia/dev/JUDI`

Comment (philippwitte) [reply src/AzureBatch/docker/tti_image/Dockerfile:10]: Not sure which scripts you mean?

Comment (philippwitte) [reply src/AzureBatch/docker/tti_image/Dockerfile:10]: I updated the script to look for the module in `/app/pysource`

Comment (mloubout) [reply src/AzureBatch/docker/tti_image/Dockerfile:10]: Oh ok cool

**Pull request merged**